liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/TextFieldDTO.kt

```diff
@@ -84,31 +84,31 @@ internal class TextFieldDTO private constructor(builder: Builder) :
         pushEvent: PushEvent
     ) {
         val label = remember(composableNode?.children) {
-            composableNode?.children?.find { it.node?.tag == TextFieldDtoFactory.label }
+            composableNode?.children?.find { it.node?.template == TextFieldDtoFactory.label }
```

Comment:
I notice `find` being used frequently for `template`. Does this only support finding the first matching element?

In the SwiftUI client we support multiple elements with the same `template`, and they will be combined into that slot. Perhaps a `filter` would be more appropriate?

Reply:
There are a few cases where several children are supported. For example, the `TopAppBar` can have multiple actions, therefore the `filter` function makes sense. 
But a `TextField` can have only one `label`. If a complex label with multiple elements is necessary (for some reason), a layout (`Box`, `Column`, `Row`, etc.) can be used to wrap these components.

Reply:
imo it should always allow multiple elements for consistency. Even if it doesn't make much sense someone may still try to do it.

If this is *possible* in Compose, then I think it should be possible in LVN too:

```kotlin
TextField(
  label = {
    Icon(...)
    Text(...)
  }
)
```

```heex
<TextField>
  <Icon template={:label} />
  <Text template={:label}> ... </Text>
</TextField>
```

Of course, if that isn't possible please disregard this.

Reply:
In the example above, how the `Icon` and the `Text` will be laid out? For a `TextField` label, just one `Text` element is expected. If you need an `Icon` and a `Text`, you will need to specify how they will be organized on the screen.
So, the solution will be:
```
<TextField>
  <Row template={:label} >
    <Icon/>
    <Text> ... </Text>
  </Row>
</TextField>
```
In Compose, when just one child is expected no layout scope is passed to the child element. But when more the one child is expected, a layout scope is used: using the example that I mentioned above, a `TopAppBar` expects more than one action, so a `RowScope` is used, which means that the action items will be laid out horizontally.  
For the cases that more than one child is expected, I'm using the `filter` function.